document/chapters/04-simd_bvh.tex

```diff
@@ -175,32 +176,34 @@ \subsection{Ray Fire Performance}\label{subsubsec:emdag_rf_performance}
-of design without the restriction of a ray tracing implementation inside the
-context of another application. The flexibility of MOAB's core design allows for the
-robust implementation of an oriented bounding box tree within this context, but
-comes with the overhead of database calls to retrieve stored information which
-can be undesirable for a high-performance system and doesn't allow MOAB to take
-advantage of some implementation optimization available in Embree. The
-vectorization of Embree's traversal through its BVH contributes greatly to its
-speed. This can be also considered a part of the design freedom allowed when
-designing an independent ray tracing system that cannot not be afforded using
-only MOAB's database interface. However, a property unique to MOAB may allow for
-an implementation with performance comparable to Embree's.
+decreasing faceting tolerance. Though MOAB is using OBBs and Embree uses AABBs,
+this indicates that both systems are capable of building efficient BVHs for a
+model with long, skinny triangles. For both MOAB and Embree, the scaling of the
+spherical case with increasing triangles is slightly worse than the HAR cylinder
+most likely because the BVH tree is unavoidably going to become deeper as more
+and more triangles exist in the model, requiring more traversal steps to reach
+leaf nodes. Finally, the slotted sphere contains many high valence regions and
+as expected it scales the worst with decreasing faceting tolerance as the
+valence of these regions increases. Due to the very similar scaling of each test
+model, it can be stated that the majority of the discrepancy in the ray fire
+timings between the two systems occurs in the traversal methods employed by both
+systems and isn't likely due to a significant difference in the quality of the
+BVH being built by either system. Changes in the way the BVH is built typically
+accounts for anywhere from 30-40\% difference in ray fire timings whereas the
+discrepancy seen here between MOAB and Embree is on average \textbf{an order of
+  magnitude better} when using Embree. To some degree, this has to do with
+Embree's freedom of design without the restriction of a ray tracing
+implementation inside the context of another application. The flexibility of
+MOAB's core design allows for the robust implementation of an oriented bounding
+box tree within this context, but comes with the overhead of database calls to
+retrieve stored information which can be undesirable for a high-performance
+system and doesn't allow MOAB to take advantage of some implementation
+optimization available in Embree. The vectorization of Embree's traversal
+through its BVH contributes greatly to its speed. This can be also considered a
+part of the design freedom allowed when designing an independent ray tracing
+system that cannot not be afforded using only MOAB's database
+interface. However, a property unique to MOAB may allow for an implementation
```

Comment:
Do you ever make it clear which property this refers to?

Reply:
I do in the direct access section, but allusions to it isn't really appropriate for this document. Replacing with a more explicit statement and reference to that section.